_posts/2019-06-30-mid-year-newsletter.md

```diff
@@ -0,0 +1,76 @@
+
+  This lesson introduces gravity models as a means for determining the probable distribution of entities across space in historical datasets. It does so through a case study of historical migration patterns.
+
+- [Analyzing Documents with TF-IDF](https://programminghistorian.org/en/lessons/analyzing-documents-with-tfidf) by Matthew J. Lavin
+
+  This lesson focuses on a foundational natural language processing and information retrieval method called Term Frequency - Inverse Document Frequency (tf-idf). This lesson explores the foundations of tf-idf, and will also introduce you to some of the questions and concepts of computationally oriented text analysis.
+
+- [Creación de corpus paralelo con LF Aligner](https://programminghistorian.org/es/lecciones/corpus-paralelo-lfaligner) por Armando Luza
+
+  Esta lección enseña a crear corpus con traducciones a distintas lenguas, de forma que queden perfectamente alineados para una mejor visualización y consulta.
+
+- [Análisis de corpus con Voyant Tools](https://programminghistorian.org/es/lecciones/analisis-voyant-tools) por Silvia Gutiérrez
+
+  En este tutorial se aprenderá cómo organizar y analizar un conjunto de textos con Voyant-Tools.
+
+- [Análisis de redes temporal](https://programminghistorian.org/es/lecciones/analisis-temporal-red) en R por Alex Breay (traducción)
+
+  Aprende a utilizar R para analizar cómo cambian las redes a lo largo del tiempo.
+
+- [Nettoyer ses données avec OpenRefine](https://programminghistorian.org/fr/lecons/nettoyer-ses-donnees-avec-openrefine) par Seth Van Hooland, Ruben Verborgh, et Max De Wilde (traduction)
+
+  Ce tutoriel se concentre sur la façon dont les chercheurs et chercheuses peuvent évaluer et agir sur l’exactitude des données.
+
+- [Introduction à la stylométrie en Python](https://programminghistorian.org/fr/lecons/introduction-a-la-stylometrie-avec-python) par François Dominic Laramée (traduction)
+
+  Ce tutoriel vous apprendra comment appliquer diverses méthodes stylométriques pour déterminer les auteurs de textes disputés. Trois méthodes sont abordées: les courbes caractéristiques de composition de Mendenhall, la méthode du khi carré de Kilgariff et, enfin, la méthode du Delta de John Burrows.
+
+
+
+## Upcoming Events 
+On July 11, Antonio Rojas Castro, Anna-Maria Sichani and Sofia Papastamkou will present their poster "Designing Multilingual Digital Pedagogy Initiatives: The Programming Historian for English, Spanish, and French speaking DH Communities" at the **DH 2019** Conference in Utrecht, The Netherlands. 
+
+Part of the team of *The Programming Historian en español* will present at the ACH 2019 Conference on July 25, in Pittsburg. Maria José Afanador-Llach, Jennifer Isasi and Antonio Rojas Castro are presenting "Retos en la producción de tutoriales de HD en contextos hispanohablantes" or "Challenges in DH lesson production in Spanish speaking contexts." 
```

Comment:
`Pittsburg` should be `Pittsburgh`

Also, maybe add links to the DH 2019 and ACH 2019 conference web pages?

Reply:
Thank you @mdlincoln for all this comments! I have made the changes and so this should be ready to go (later than I had planned to)